circle.yml

```diff
@@ -410,168 +410,174 @@ jobs:
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: chrome
-          chunk: 6
+          chunk: "6"
 
   "server-e2e-tests-chrome-7":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: chrome
-          chunk: 7
+          chunk: "7"
 
   "server-e2e-tests-chrome-8":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: chrome
-          chunk: 8
+          chunk: "8"
 
   "server-e2e-tests-electron-1":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: electron
-          chunk: 1
+          chunk: "1"
 
   "server-e2e-tests-electron-2":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: electron
-          chunk: 2
+          chunk: "2"
 
   "server-e2e-tests-electron-3":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: electron
-          chunk: 3
+          chunk: "3"
 
   "server-e2e-tests-electron-4":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: electron
-          chunk: 4
+          chunk: "4"
 
   "server-e2e-tests-electron-5":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: electron
-          chunk: 5
+          chunk: "5"
 
   "server-e2e-tests-electron-6":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: electron
-          chunk: 6
+          chunk: "6"
 
   "server-e2e-tests-electron-7":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: electron
-          chunk: 7
+          chunk: "7"
 
   "server-e2e-tests-electron-8":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: electron
-          chunk: 8
+          chunk: "8"
+
+  "server-e2e-tests-non-root":
+    <<: *defaults
+    steps:
+      - run-e2e-tests:
+          chunk: non_root
 
   "server-e2e-tests-firefox-1":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: firefox
-          chunk: 1
+          chunk: "1"
 
   "server-e2e-tests-firefox-2":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: firefox
-          chunk: 2
+          chunk: "2"
 
   "server-e2e-tests-firefox-3":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: firefox
-          chunk: 3
+          chunk: "3"
 
   "server-e2e-tests-firefox-4":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: firefox
-          chunk: 4
+          chunk: "4"
 
   "server-e2e-tests-firefox-5":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: firefox
-          chunk: 5
+          chunk: "5"
 
   "server-e2e-tests-firefox-6":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: firefox
-          chunk: 6
+          chunk: "6"
 
   "server-e2e-tests-firefox-7":
     <<: *defaults
     steps:
       - run-e2e-tests:
           browser: firefox
-          chunk: 7
+          chunk: "7"
```

Comment:
why change chunking to strings?

Reply:
added "non_root" chunk, and circleci has strict types for command parameters, so need to force these to be strings or else circleci complains about supplying an integer to a string parameter